src/Tribe/Views/V2/View.php

```diff
@@ -603,6 +603,10 @@ public function get_url( $canonical = false ) {
 			'tribe-bar-search' => $this->context->get( 'keyword', '' ),
 		];
 
+		if ( tribe_is_truthy( $this->context->get( 'featured', false ) ) ) {
```

Comment:
Should we assign the results of this `tribe_is_truthy()` so we don't need to execute that and the `context->get()` twice? Like:

```php
if ( $is_featured = tribe_is_truthy( $this->context->get( 'featured', false ) ) ) {
```